app/controllers/users_answers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class UsersAnswersController < ApplicationController
+  def index
+    user_answers = Answers.find_by_user_id(params[:id])
```

Comment:
```suggestion
    user_answers = Answer.find_by_user_id(params[:id])
```

The model is singular, or am I missing something?

However, if you are going to return an array, why not this?

```suggestion
    render json: Answer.where(user_id: params[:id])
```

Reply:
Yes! Thanks